diff --git a/node_modules/elliptic/lib/elliptic/curves.js b/node_modules/elliptic/lib/elliptic/curves.js
index 63d8655..af25b47 100644
--- a/node_modules/elliptic/lib/elliptic/curves.js
+++ b/node_modules/elliptic/lib/elliptic/curves.js
@@ -204,3 +204,20 @@ defineCurve('secp256k1', {
     pre
   ]
 });
+
+defineCurve('secp160r1', {
+  type: 'short',
+  prime: 'r160',
+  p: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7FFFFFFF',
+  a: 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7FFFFFFC',
+  b: '1C97BEFC54BD7A8B65ACF89F81D4D4ADC565FA45',
+  n: '0100000000000000000001F4C8F927AED3CA752257',
+  h: '1',
+  hash: hash.sha256,
+
+  gRed: false,
+  g: [
+    '4A96B5688EF573284664698968C38BB913CBFC82',
+    '23A628553168947D59DCC912042351377AC5FB32',
+  ],
+});
